ui/DesignSystem/Component/Header/Large.svelte

```diff
@@ -10,6 +10,15 @@
   export let entity = null;
   export let variant = null; // profile | org
 
+  const name =
+    variant === "profile" && entity.registered
+      ? entity.registered
+      : variant === "profile" && !entity.registered
+      ? entity.metadata.handle
+      : variant === "org"
+      ? entity.id
+      : null;
```

Comment:
I feel like nested if's would be more legible here.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/758/commits/0b9867696a1ef5ce4d154ef1d30462fa65af6006